samples/drivers/seu/README.rst

```diff
@@ -0,0 +1,56 @@
+.. _seu_sample:
```

Comment:
you may use `.. zephyr:code-sample::` directive here, especially once you'll have define a doxygen group for the API as this will allow cross-referencing the sample automatically from the API Reference page.

Reply:
changes updated